libraries/client/src/main/java/org/holochain/androidserviceruntime/client/Parcelables.kt

```diff
@@ -111,3 +111,15 @@ data class ZomeCallUnsignedFfiParcel(
 data class ZomeCallFfiParcel(
     val inner: ZomeCallFfi,
 ) : Parcelable
+
+@Parcelize
+@TypeParceler<AdminBinderUnauthorizedException, AdminBinderUnauthorizedExceptionParceler>
+data class AdminBinderUnauthorizedExceptionParcel(
```

Comment:
Types to send our custom exceptions over the client-service ipc connection. We can't throw them as normal between processes, so instead we callback with the exception data.